migrations/20180510172759_data_import_permission_name_change.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+
+class DataImportPermissionNameChange extends AbstractMigration
```

Comment:
👍

Reply:
... I have a concern as I thought more about this... this is a breaking migration ie. once we deploy the code users will lose the data import permission until the migration runs.
Trying to think if theres a un-complicated way to avoid that :/